src/test/ui/const-generics/const-argument-cross-crate-mismatch.stderr

```diff
@@ -0,0 +1,21 @@
+error[E0308]: mismatched types
+  --> $DIR/const-argument-cross-crate-mismatch.rs:6:41
+   |
+LL |     let _ = const_generic_lib::function(const_generic_lib::Struct([0u8, 1u8]));
+   |                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `3usize`, found `2usize`
+   |
+   = note: expected type `const_generic_lib::Struct<3usize>`
+              found type `const_generic_lib::Struct<_: usize>`
```

Comment:
@varkor What's up with `<_: usize>` here?

Reply:
Not sure, as it seems to know what the value is. We can open another issue for it.

Reply:
Opened https://github.com/rust-lang/rust/issues/65380.